### PR TITLE
Add stale GitHub action.

### PR DESCRIPTION
**Pull request by jezdez**
This should help with handling old issues and PRs.

There is some research around that as well: https://www.researchgate.net/publication/331718820_Should_I_Stale_or_Should_I_Close_An_Analysis_of_a_Bot_That_Closes_Abandoned_Issues_and_Pull_Requests

### CODE REVIEW

**Review by dbast — APPROVED** [review]

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            Hi there, thank you for your contribution to Conda!
+
+            This issue has been automatically marked as "stale" because
+            it has not had recent activity. It will be closed automatically
+            if no further activity occurs.
+
+            If you would like this issue to remain open please:
+
+              1. Verify that you can still reproduce the issue in the latest version of Conda
+              1. Comment that the issue is still reproducible and include:
```
Comment (jaimergp) [inline .github/workflows/stale.yml:34]:
```suggestion
              2. Comment that the issue is still reproducible and include:
```

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+            please reopen the issue there again.
+
+          # Comment on the staled issues while closed
+          close-issue-message: >
+            Hi again!
+
+            This issue has been closed since it has not had recent activity.
+            Please don't hesitate to leave a comment if that was done prematurely.
+
+            Thank you for your contribution
```
Comment (jaimergp) [inline .github/workflows/stale.yml:53]:
```suggestion
            Thank you for your contribution.
```

**Comment by jezdez** [comment]
Thanks @jaimergp and @dbast!

**Review by chenghlee — APPROVED** [review]

**Review by csoja — COMMENTED** [review]
The wording on these changes sounds good to me.  Thank you for adding it!

**Pull request merged**